transfer_nlp/plugins/config.py

```diff
@@ -148,6 +147,155 @@ def recursive_replace(dico: Dict):
     recursive_replace(dico=dico)
 
 
+class ExperimentConfig2:
```

Comment:
why do we have both ExperimentConfig and ExperimentConfig2? I assume the old code should be deleted?